CONTRIBUTING.md

```diff
@@ -72,71 +92,79 @@ By making a contribution to this project, I certify that:
-  * `hotfix/` - bug fixes that got through and need to be squashed
-  * `release/` - for releases
-  * `feature/` - update to or new functionality
+  * `hotfix/` - [bug fixes](#bug-repott) for bugs got through and need to be squashed
+  * `feature/` - update to or [new functionality](#functionality-requests)
 
-## Creating a New Version
+### Time to Merge
 
-Versioning is done through [SEMVER](http://semver.org/). When issuing a new version, create a [release branch](#branching-model) with the version's name in to the `master` branch. The release branch should bump the version number accordingly and [update the changelog](#maintaining-the-changelog). Once merged, create a new tag with `v` prefixed with the version's name from that branch.
+In order for a pull request to be merged, it must meet the following criteria:
 
-For instance, when creating version `1.1.0`, start by creating a release branch `release/1.1.0` that includes the version bump and changelog updates, merge it in to `master` and create a tag `v1.1.0` from branch `release/1.1.0`.
+* The author(s) of the code have signed off on the [Developer Certificate of Origin](#developer-certificate-of-origin) for this pull request
+* All required integrations have passed
+* The code has been reviewed by at least one non-authoring member of the [@punchcard-cms/maintainers](https://github.com/orgs/punchcard-cms/teams/maintainers) GitHub team and all comments from the review have been addressed. This may include comments related to our [tips for submitting great PRs](#tips-for-submitting-great-prs).
 
-### Maintaining the Changelog
+## Creating a Release
 
-The Changelog should have a list of changes made for each version. They should be organized so additions come first, changes come second, and deletions come third. Version numbers should be 2nd level headers with the `v` in front (like a tag) and the date of the version's most recent update should be underneath in italics.
+Versioning is done through [SEMVER](http://semver.org/). Whenever new code is merged in to the `master` branch, a new semantic release will automatically be created on NPM and in the project's GitHub Releases. The GitHub Release will also include a changelog based off of the commit messages of that release. The release version and changelog are determined by the [punchcard-semantic-release](https://github.com/punchcard-cms/semantic-release) module, and released using the [semantic-release](https://github.com/semantic-release/semantic-release) module based off of the [emoji](#emoji-cheatsheet) used in the commit messages for each commit. The following emoji correspond to different semantic versions, all other emoji are considered non-semantic:
 
-Changelog messages do not need to cover each individual commit made, but rather should have individual summaries of the changes made. Changelog messages should be written in the same style as [commit messages](#git-commit-messages).
+**Major Version**
```

Comment:
This only covers nine of the 12 emoji. Should the other missing ones be in `In-progress release` too?